UnitsNet/CustomCode/Quantity.cs

```diff
@@ -7,7 +7,7 @@ namespace UnitsNet
 {
     public partial class Quantity
     {
-        private static readonly Lazy<QuantityInfo[]> InfosLazy;
+        private static Lazy<QuantityInfo[]> InfosLazy { get; }
```

Comment:
Same here, a static prop will never be part of an interface either, so I don't think this change adds value.